integration_tests/__tests__/start.test.js

```diff
@@ -18,17 +18,32 @@ const TEST_PROJECT_DIR = path.resolve(
   '../../fixtures/react-native-with-haul'
 );
 
-beforeAll(() => run('yarn --mutex network', TEST_PROJECT_DIR));
-beforeEach(() => cleanup(TEMP_DIR));
-afterEach(() => cleanup(TEMP_DIR));
+let haul;
+beforeAll(done => {
+  run('yarn --mutex network', TEST_PROJECT_DIR);
+  haul = runHaul(TEST_PROJECT_DIR, ['start']);
+  const messageBuffer = [];
 
-test('start command displays "Select platform" message', () => {
-  writeFiles(TEMP_DIR, {
-    'webpack.haul.js': '{}',
-  });
+  haul.stdout.on('data', data => {
+    const message = stripAnsi(data.toString()).trim();
+
+    if (message.length > 0) {
+      messageBuffer.push(message);
+    }
 
-  const { stdout } = runHaulSync(TEMP_DIR, ['start']);
-  expect(stripAnsi(stdout).trim()).toMatchSnapshot();
+    if (message.match(/ERROR/g)) {
+      done.fail(message);
```

Comment:
Can't we listen to Haul's events instead of matching stdout?

Reply:
We can, but we need to change the way we run haul.

Reply:
Would be great to do it as a followup. Feel free to disable this test for the time being.